modules/ROOT/pages/update-acm.adoc

```diff
@@ -47,6 +47,24 @@ Attempting to upgrade the package without this feature enabled causes the proces
 .. Click the Delete button.
 .. Click the Confirm delete button.
 .. Continue with normal package installation.
+. If you are upgrading from a package version before 1.74, follow these steps:
```

Comment:
At some point, consider whether you should take this subsubtask out of the container and make it a separate task.

Reply:
Good point. These instructions need some better organization.